website/src/docs/redux.md

```diff
@@ -15,19 +15,17 @@ Uppy supports the popular [Redux](https://redux.js.org/) state management librar
 You can tell Uppy to use your app’s Redux store for its files and UI state. Please check out [Custom Stores](/docs/stores/) for more information on that. Here’s an example to give you a sense of how this works:
 
 ```js
-import { createStore } from 'redux'
-import createReduxStore from '@uppy/store-redux'
+import Uppy from '@uppy/core'
+import ReduxStore from '@uppy/redux-store'
```

Comment:
The name of the package is backward, isn't it?
```suggestion
import ReduxStore  from '@uppy/store-redux'
```
Also, we should try to have our examples ESM ready (I'm not sure `import * as`  is going to work well with the current CJS implementation, but it'd be the cleanest to use).
```suggestion
import * as ReduxStore  from '@uppy/store-redux'
```
```suggestion
import { ReduxStore, reducer }  from '@uppy/store-redux'
```